presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -77,6 +78,7 @@ public LdapAuthenticator(LdapConfig ldapConfig)
         this.userBaseDistinguishedName = Optional.ofNullable(ldapConfig.getUserBaseDistinguishedName());
         this.bindDistinguishedName = Optional.ofNullable(ldapConfig.getBindDistingushedName());
         this.bindPassword = Optional.ofNullable(ldapConfig.getBindPassword());
+        this.ignoreReferrals = Optional.ofNullable(ldapConfig.isIgnoreReferralsEnabled());
```

Comment:
`ldapConfig.isIgnoreReferralsEnabled()` is `boolean` (primitive), so you don't need `Optional`. Make the field `boolean`

```suggestion
        this.ignoreReferrals = ldapConfig.isIgnoreReferralsEnabled();
```